CHANGELOG.md

```diff
@@ -272,7 +274,8 @@ PLEASE REMEMBER TO CHANGE THE '..master' WITH AN ACTUAL TAG in GITHUB LINK.
 
 * [GitHub commits](https://github.com/google/jax/compare/jax-v0.1.77...jax-v0.2.0).
 * Improvements:
-  * Omnistaging on by default. See {jax-issue}`#3370`.
+  * Omnistaging on by default. See {jax-issue}`#3370` and
```

Comment:
I think the file link might be to prng.md rather than omnistaging.md.

Reply:
Done, thanks!